torch/testing/_internal/opinfo/utils.py

```diff
@@ -103,7 +103,7 @@ def dtypes_dispatch_hint(dtypes):
 
     # CUDA is not available, dtypes will be empty.
     if len(dtypes) == 0:
-        return return_type((), str(tuple()))
+        return return_type((), str(()))
```

Comment:
`str(tuple())` is always `"()"`, isn't it?
```suggestion
        return return_type((), "()")
```